CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+## [1.8.1]
```

Comment:
This should be a link in the bottom that links to https://github.com/giantswarm/net-exporter/compare/v1.8.0..v1.8.1